parsl/executors/high_throughput/interchange.py

```diff
@@ -53,6 +53,7 @@ def __init__(self,
                  logging_level: int,
                  poll_period: int,
                  cert_dir: Optional[str],
+                 manager_selector: Optional[ManagerSelectorBase] = None,
```

Comment:
have a look at PR #3465 and reconsider this line in that context

Reply:
Will remove this default value.